webui/src/pages/repositories/repository/pulls/pulls.jsx

```diff
@@ -0,0 +1,159 @@
+import React, {useEffect, useState} from "react";
+import {useOutletContext} from "react-router-dom";
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+import Alert from "react-bootstrap/Alert";
+import {Tab, Tabs} from "react-bootstrap";
+import Button from "react-bootstrap/Button";
+import TimeAgo from "react-timeago";
+
+import {ActionGroup, AlertError, Loading, PrefixSearchWidget, RefreshButton} from "../../../../lib/components/controls";
+import {pulls} from "../../../../lib/api";
+import {useRefs} from "../../../../lib/hooks/repo";
+import {useAPIWithPagination} from "../../../../lib/hooks/api";
+import {Paginator} from "../../../../lib/components/pagination";
+import {useRouter} from "../../../../lib/hooks/router";
+import {RepoError} from "../error";
+import {Link} from "../../../../lib/components/nav";
+
+
+const PullWidget = ({repo, pull}) => {
+    const authorLink =
+        <Link href={{
+            // TODO: to where this link should lead?
+            pathname: '/repositories/:repoId/user/:userId',
+            params: {repoId: repo.id, userId: pull.author}
+        }}>
+            {pull.author}
+        </Link>;
+    return (
+        <ListGroup.Item className="pull-row">
```

Comment:
Look at the example of commits tab, it should look the same IMO.
Font size, alignment hyperlink marking and the visuals (images and colors) should have the same style.
This tab should look and feel like the other tabs